libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -635,14 +635,14 @@ public void run() {
                         getContent();
                     } catch (IllegalEditorStateException e) {
                         AppLog.e(T.EDITOR, "toggleHtmlMode: unable to get title or content");
-                        getActivity().runOnUiThread(new Runnable() {
-                            @Override
-                            public void run() {
-                                toggleButton.setChecked(false);
```

Comment:
Hey @khaykov , can you share some background on removing the toggling back to `false` here? In particular, is the removal on purpose?

Reply:
Sure. In my understanding, at the time this code is executed fragment is already detached from activity and does not have UI (onDestroyView() was called). If and when fragment will be recreated it will be in the default state with the button unchecked.

Reply:
Ah, I see, makes sense, thanks!

Going a step further, `IllegalEditorStateException` has a pretty general name, implying that there might be various reasons the Editor might be in an illegal state. But at the moment, it is only thrown when the fragment is not added. I think this is rather confusing (hence my original question :) ) so, what do you think about renaming the exception class to smth like `FragmentNotAdded` perhaps?

Reply:
This is a good idea :) Renamed to something more descriptive.